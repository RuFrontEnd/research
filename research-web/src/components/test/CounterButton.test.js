import { shallow } from "enzyme";
import CounterButton from "components/test/CounterButton";

it("expect to render CounterButton component", () => {
  const mockColor = "red";
  const $counterButton = shallow(<CounterButton color={mockColor} />);
  expect($counterButton).toMatchSnapshot();
});

it("expect to click CounterButton component", () => {
  const mockColor = "red";
  const $counterButton = shallow(<CounterButton color={mockColor} />);
  expect($counterButton).toMatchSnapshot();
});
