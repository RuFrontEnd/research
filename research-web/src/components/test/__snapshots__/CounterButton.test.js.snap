// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to click CounterButton component 1`] = `
<button
  color="red"
  onClick={[Function]}
>
  ++
</button>
`;

exports[`expect to render CounterButton component 1`] = `
<button
  color="red"
  onClick={[Function]}
>
  ++
</button>
`;
